src/vgg_model.py

```diff
@@ -0,0 +1,95 @@
+import os
+from pathlib import Path
+import tensorflow as tf
+from keras.applications.vgg16 import VGG16
+from keras.preprocessing import image
+from keras.applications.vgg16 import preprocess_input
+from keras.models import Model
+import numpy as np
+from vgg_model import VGG16
+from scipy import stats
+from sklearn.metrics.pairwise import cosine_similarity
+
+
+class VGG16_model():
+    def __init__(self,image_dataset_path,dataset_path):
+        self.image_path = image_dataset_path
+        self.dataset = dataset_path
+    
+
+    def load_image(self):
+        '''function: to path the input images directory and create list of image name
+
+                parameter:
+                None
+
+                Return:
+                list of images
+        '''
+        images=[]
+        directory = Path(self.image_path)
+        file_path = os.listdir(directory)
+        for image in file_path:
+            images.append(image)
+        images.sort()
+        #if condition returns False, AssertionError is raised:
```

Comment:
Looks like it needs to be removed.

Reply:
I have removed it. Check it once again. @raiResham